challenges/print_level_order/print_level_order.py

```diff
@@ -19,12 +19,11 @@ def recurse(nodelist):
 
         string = ' '.join(printlist)
         all_strings.append(string)
-        print(string)
 
         if len(new_list):
```

Comment:
It is not necessary to explicitly check the length of a list for a conditional. An empty list is falsey and and a filled list is truthy.